src/components/tables/DataTable/table/DataTable.tsx

```diff
@@ -3,6 +3,7 @@
 |* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import type * as React from 'react';
+import { Children, isValidElement, cloneElement, useRef, useState, useEffect } from 'react';
```

Comment:
Use a `React` namespace for these.

Change `import type * as React from 'react'` to `import * as React from 'react'` and then use `React.` for all these imports.